content/en/platform/corda/5.0/deploying-operating/deploying/deploying.md

```diff
@@ -310,6 +323,42 @@ username:
   value: <USERNAME>
 ```
 
+### Encryption
+
+The Corda configuration system allows for any string configuration value to be marked as “secret”. This includes values passed dynamically using the REST API and also those defined in a manual deployment configuration. For more information see, [Configuration Secrets]({{< relref "../config/secrets.md" >}}).
+
+#### Default Secrets Service
+
+The [Corda default secrets lookup service]({{< relref "../config/secrets.md#default-secrets-service" >}}) uses a salt and passphrase specified in the deployment configuration. Specify these as follows:
+
+```yaml
+config:
+   encryption:
+      salt: <SALT>
+      passphrase: <PASSPHRASE>
+```
+
+#### External Secrets Service {{< enterprise-icon >}}
+
+To configure Corda Enterprise to connect to a running [HashiCorp Vault instance]({{< relref "../config/secrets.md#external-secrets-service" >}}), add the following:
+
+```yaml
+config:
+  vault:
+    url: "<vault-URL>"
+    token: "<vault-token>"
```

Comment:
Shame that `token` doesn't also support the `value`/`valueFrom` syntax but I didn't pick up on it at the time and it's too late now.